test/functional/test_framework/test_framework.py

```diff
@@ -2126,4 +2126,4 @@ def test():
                 if "verified_proregtx_hash" in p and p["verified_proregtx_hash"] != "":
                     c += 1
             return c >= count
-        wait_until_helper(test, timeout=timeout)
+        self.wait_until(test, timeout=timeout)
```

Comment:
I think 2 more usages in p2p_quorum_data.py can be updated too:

```
    wait_until_helper(lambda: get_score() == expected_score, timeout=6)
    wait_until_helper(lambda: get_id() is not None, timeout=10)
```

Reply:
Can't refactor these without moving func def-s into `QuorumDataMessagesTest` but there are some `wait_until_helper `s in `feature_governance.py` and `feature_llmq_signing.py` that could be refactored too.

Reply:
just clarifying, no action needed here?

Reply:
> just clarifying, no action needed here?

no action is needed for my comment about `p2p_quorum_data.py` `p2p_quorum_data.py` requires intensive refactoring by moving many functions inside class, only if you are will do to it.

But as Udjin noticed, once `do_assert` is added to `self.wait_until`'s helper **it can be applied to `feature_governance.py` and `feature_llmq_signing.py`**.